Update Requirements.md

Germano123 opened the pull request:
add gamification and tokenization for user in course payment section and add events section.

Germano123 committed:
Update Requirements.md

add gamification and tokenization for user in course payment section and add events section.

tolstenko commented on Requirements.md:
```diff
@@ -69,6 +70,17 @@ Game Marketplace:
 
+### 2.4 Events
+
+Weekly events:
```
Don't use weekly. It could be any time window.

tolstenko commented on Requirements.md:
```diff
@@ -69,6 +70,17 @@ Game Marketplace:
+Weekly events:
+
+- A roll of events that take place every week created by the users.
+- An user can 'buy' with platform tokens any event from anyone that has ever submited that event.
```
Events can be open to everyone. Improve the writing to reflect it

tolstenko commented on Requirements.md:
```diff
@@ -69,6 +70,17 @@ Game Marketplace:
+
+Game Jams:
+
+- Events that can be organized inside the community
```
Add more information about game jams

tolstenko reviewed (commented):
- Needs rework to make events with different frequency.
- Add more details to game jam management system

Germano123 committed:
Update Requirements.md

Expand details in section 2.4 Events and add section 2.5 Blog to increment a well documented feature, as we reopened the issue.

tolstenko commented on Requirements.md:
```diff
@@ -69,6 +70,41 @@ Game Marketplace:
+- Users can submit events for evaluation, including workshops, lectures, or other activities related to game development.
+- Once approved, events are available for the community to view, attend, and participate in.
+
+Elected Meetings:
```
Rename "Elected Meetings" to "ChoiceCast"

tolstenko commented on Requirements.md:
```diff
@@ -69,6 +70,41 @@ Game Marketplace:
+
+Elected Meetings:
+
+- Every week, the community votes on one event to become the "Elected Meeting," a highlighted event in the weekly calendar.
```
Don't use "every week", this puts pressure on us. rework this phrase to make it a routine but not every week.

tolstenko reviewed (commented):
Read notes inline. Don't use "event" work too much. Be more precise. If in doubt, ask chatgpt to help you improve the writing.

Germano123 committed:
Update Requirements.md

tolstenko commented on Requirements.md:
```diff
@@ -69,6 +70,42 @@ Game Marketplace:
+General Meetings:
+
+- Meetings can be created by any logged-in user on the platform at any time.
+- Users can submit meetings for evaluation, including workshops, lectures, or other activities related to game development.
```
"- Users can submit meetings for evaluation, including workshops, lectures, or other activities related to game development."
I think it is mis placed. it should be on choice cast, right?

tolstenko commented on Requirements.md:
```diff
@@ -69,6 +70,42 @@ Game Marketplace:
+
+- Meetings can be created by any logged-in user on the platform at any time.
+- Users can submit meetings for evaluation, including workshops, lectures, or other activities related to game development.
+- Once approved, meetings are available for the community to view, attend, and participate in.
```
same thing here

tolstenko commented on Requirements.md:
```diff
@@ -69,6 +70,42 @@ Game Marketplace:
+
+Choice Casts:
+
+- Every week, the community votes on one event to become the "Choice Cast", a highlighted meeting in the calendar.
```
For the third time: Don't fix the time frequency to weekly based. It can be anything, and it will be more sane to ourselves.

tolstenko reviewed (commented):
read comments inline

Germano123 committed:
Update Requirements.md

tolstenko reviewed (commented):
reviews in line

Germano123 committed:
Update Requirements.md

Germano123 commented:
Everythin reviewed and updated with corrections.

tolstenko commented on Requirements.md:
```diff
@@ -75,8 +75,8 @@ Game Marketplace:
 - Meetings can be created by any logged-in user on the platform at any time.
-- Users can submit meetings for evaluation, including workshops, lectures, or other activities related to game development.
-- Once approved, meetings are available for the community to view, attend, and participate in.
+- Users can submit meetings any meeting including workshops, lectures, or other activities related to game development.
```
It doesn't make sense this phrase. Please rewrite it or chatgpt to improve the quality of your writtng.

Germano123 replied:
ok, already doing it.

tolstenko reviewed (commented):
Please improve the wording. Your phrase does not make sense.

Germano123 committed:
Update Requirements.md

Germano123 committed:
docs: improved writing in requeriments.md

tolstenko reviewed (commented):
LGTM
